plugins/generic/pln/ping.dtd

```diff
@@ -14,10 +14,19 @@
 <!ELEMENT ojsInfo (release)>
 <!ELEMENT release (#PCDATA)>
 
-<!ELEMENT pluginInfo (release, releaseDate, installed, current)>
+<!ELEMENT pluginInfo (release, releaseDate, installed, current, terms)>
 <!ELEMENT releaseDate (#PCDATA)>
 <!ELEMENT installed (#PCDATA)>
 <!ELEMENT current (#PCDATA)>
+<!ELEMENT terms (term*)>
+		<!ATTLIST terms
+				termsAccepted CDATA #REQUIRED>
```

Comment:
From the code, this looks like a controlled vocabulary of yes|no -- better to explicitly specify that here.